tests/python/gaia-ui-tests/gaiatest/tests/brick/test_brick_verification.py

```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marionette_driver import expected, By, Wait
+from gaiatest import GaiaTestCase
+
+
+class TestBrickVerification(GaiaTestCase):
+
+    def test_wait_for_homescreen(self):
+        homescreen_locator = (By.CSS_SELECTOR, '#icons')
+        self.apps.switch_to_displayed_app()
+        Wait(self.marionette).until(expected.element_displayed(*homescreen_locator))
```

Comment:
If you wanted to make this even lighter, you could just do:

```python
self.assertEqual(self.apps.displayed_app.name, 'Homescreen')
```

Then there's no need for the switch, the locator or the wait (the setUp already waits for the homescreen to be ready).